I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/map_screen.rpy", line 59, in script
    if firstMart == True:
  File "game/map_screen.rpy", line 66, in <module>
    elif KRS4 == True:
NameError: name 'KRS4' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/map_screen.rpy", line 59, in script
    if firstMart == True:
  File "C:\Users\ASUS\Downloads\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\ast.py", line 2115, in execute
    if renpy.python.py_eval(condition):
  File "C:\Users\ASUS\Downloads\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\python.py", line 1092, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\ASUS\Downloads\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/map_screen.rpy", line 66, in <module>
    elif KRS4 == True:
NameError: name 'KRS4' is not defined

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090405
Simulasi Kehidupan Kuliah 1.0
Sun Jan 29 20:27:22 2023
